bin.src/runIsr.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+#
+# LSST Data Management System
+# Copyright 2008, 2009, 2010 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.    See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+from lsst.ip.isr.isrTask import IsrWrapperTask
+
+IsrWrapperTask.parseAndRun()
```

Comment:
Might just be me, but I would expect this to tun `RunIsrTask` given the name of the script is `runIsr`.    Remember all the eye-rolling over  `bin/measureCoaddSources.py` calling 
`MeasureMergedCoaddSourcesTask.parseAndRun()` and the like in pipe_tasks

Reply:
I am open to different solutions, but I don't see one that doesn't involve either 1) making separate runIsr_obsX.py scripts for each obs_package; or 2) making the current `RunIsrTask` into `RunIsrRealTask` or something equivalent.

Reply:
Is there already a `RunIsrTask`???

Reply:
The pattern now should be `IsrTask` + camera-specific IsrTask's like `DecamNullIsrTask`. To my knowledge, there is no `RunIsrTask`.   I'm going offline for the night. I don't care enough for you to change it to `RunIsrTask`,  but be aware one of the PipelineTask precursor/refactoring jobs that @natelust going to work on is making the multiband script names match their Task names in pipe_tasks.